CONTRIBUTING.md

```diff
@@ -6,20 +6,20 @@ We operate the "Fork & Pull" model explained at
 
 https://help.github.com/articles/using-pull-requests
 
-You should fork the project into your own repo, create a topic branch
-there and then make one or more pull requests back to the OpenStreetMap repository.
-Your pull requests will then be reviewed and discussed.
+You should fork the project into your own repo, create a topic branch there and
+then make one or more pull requests back to the OpenStreetMap repository. Your
+pull requests will then be reviewed and discussed.
 
 ## History
 
-To understand the osm2pgsql code, it helps to know some history on it. Osm2pgsql
-was written in C in 2007 as a port of an older Python utility. In 2014 it was
-ported to C++ by MapQuest and the last C version was released as 0.86.0. In it's
-time, it has had varying contribution activity, including times with no
-maintainer or active developers.
+To understand the osm2pgsql code, it helps to know some history on it.
+Osm2pgsql was written in C in 2007 as a port of an older Python utility. In
+2014 it was ported to C++ by MapQuest and the last C version was released as
+0.86.0. In it's time, it has had varying contribution activity, including times
+with no maintainer or active developers.
 
-Parts of the codebase still clearly show their C origin and could use rewriting
-in modern C++, making use of data structures in the standard library.
+There are still a few parts that show their C origin, but most of the code is
+now modern C++. We are currently targeting C++17.
```

Comment:
Far too modest. "Very few parts of the code now show their C origin, most has been transformed to modern C++."

Reply:
Fixed